.appveyor.yml

```diff
@@ -0,0 +1,36 @@
+# AppVeyor file
+# http://www.appveyor.com/docs/appveyor-yml
+
+# Build version format
+version: "{build}"
+
+# Do not build feature branch with open Pull Requests
+skip_branch_with_pr: true
```

Comment:
Why not?

Reply:
ping @limonte

Reply:
Having a build for the branch is useful if there's some error in the merged version, and you want to know if it was introduced by the merge or if it was already there in the branch exactly as you pushed it.